lib/rubocop/cli.rb

```diff
@@ -3,7 +3,7 @@
 module RuboCop
   # The CLI is a class responsible of handling all the command line interface
   # logic.
-  class CLI
+  class CLI # rubocop:disable Metrics/ClassLength
```

Comment:
As that's the only class here I'd probably place this at the beginning of the source file.